db/migrate/20170222214902_add_initiator_to_service.rb

```diff
@@ -0,0 +1,14 @@
+class AddInitiatorToService < ActiveRecord::Migration[5.0]
+  class Service < ActiveRecord::Base; end
+  def up
+    add_column :services, :initiator, :string
+    change_column_comment :services, :initiator, "Record who initiated the service creation user|control|alert. It helps the UI display the services based on categories"
+    say_with_time('Adding initiator to Service') do
```

Comment:
This is not meaningful to someone watching the db:migrate.  Perhaps `"Updating existing services to 'user' initiator"` ?